Packs/SecurityAdvisor/Integrations/SecurityAdvisor/integration-SecurityAdvisor.yml

```diff
@@ -0,0 +1,207 @@
+        """
+
+        def http_request_coachuser(self, data):
+            """
+            calls coach user api
+            """
+            response_data_json = self._http_request(
+                method='POST',
+                url_suffix=URL_SUFFIX_COACH_USER,
+                json_data=data,
+                data=data,
+            )
+            return response_data_json
+
+
+    def coach_end_user_command(client, args):
+        """
+        Returns Coaching status of user
+
+        Args:
+            client: SecurityAdvisor client
+            args: all command arguments
+
+        Returns:
+            json version of coaching status for user
+            readable_output: This will be presented in Warroom - should be in markdown syntax - human readable
+            outputs: Dictionary/JSON - saved in incident context in order to be used as input for other tasks in the
+                     playbook
+            raw_response: Used for debugging/troubleshooting purposes - will be shown only if the command executed with
+                          raw-response=true
+        """
+        user = args.get('user')
+        context = args.get('context')
+        data = json.dumps({"username": user, "context": context})
+        result = client.http_request_coachuser(data)
+
+        contxt = makehash()
+        contxt['user'] = user
+        contxt['context'] = context
+        contxt['message'] = result['message']
+        contxt['coaching_status'] = result['coaching_status']
+        contxt['coaching_score'] = result['coaching_score']
+        contxt['coaching_date'] = result['coaching_date']
+        outputs = ({
+            'SecurityAdvisor.CoachUser(val.user == obj.user && val.context == obj.context)': contxt,
+        })
+
+        readable_output = tableToMarkdown("Coaching Status", [contxt])
+
+        return (
+            readable_output,
+            outputs,
+            result  # raw response - the original response
+        )
+
+
+    def test_module(client):
+        """Test Module when testing integration"""
+        data = json.dumps({
+            "username": "track@securityadvisor.io",
+            "context": "malware"
+        })
+        client.http_request_coachuser(data)
+
+        return 'ok'
+
+
+    ''' EXECUTION '''
+
+
+
+    def main():
+        """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+        """
+        base_url = demisto.params().get('url', 'https://www.securityadvisor.io/')
+        proxy = demisto.params().get('proxy')
+        api_key = demisto.params().get('apikey')
+        verify_certificate = not demisto.params().get('insecure', False)
+        if not demisto.params().get('proxy', False):
+            try:
+                del os.environ['HTTP_PROXY']
+                del os.environ['HTTPS_PROXY']
+                del os.environ['http_proxy']
+                del os.environ['https_proxy']
+            except KeyError:
+                pass
```

Comment:
@Bargenish thanks for the feedback. Are you suggesting i remove Packs/SecurityAdvisor/Integrations/SecurityAdvisor/SecurityAdvisor.yml? and keep integration-SecurityAdvisor.yml? Or remove integration-SecurityAdvisor.yml since that will generated with split-sdk command?

Reply:
@Bargenish committed with both the YAML files.